daemon/daemon.go

```diff
@@ -109,6 +111,8 @@ type Daemon struct {
 	defaultLogConfig runconfig.LogConfig
 	RegistryService  *registry.Service
 	EventsService    *events.Events
+	uidMaps          []idtools.IDMap
```

Comment:
I tried to share it! :) some folks didn't like `libcontainer/configs` being used in a `pkg/` subdirectory and asked me to have a shadow struct in docker/docker